### PR TITLE
Correct sentence describing the password usage

### PR DESCRIPTION
**Pull request by yahiheb**
As discussed with @lontivero the password doesn't encrypt the mnemonic recovery words but rather it encrypts the private key of the extended private key, and the encrypted private key is stored on the computer.

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -98,11 +96,11 @@ You can also easily reach it from inside Wasabi: `File > Open > Data Folder`.
 Although you can backup your private keys with the mnemonic words and password, this is only a last resort recovery.
 If you want to also secure your address labels, the anonset and additional metadata, then you can do a digital backup.
 Simply copy the `WalletBackups` folder with the `wallet.json` files from your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) onto suitable hardware, for example an encrypted USB stick.
-Note that this file has the encrypted private keys, meaning that you only need the password to spend the bitcoin.
+Note that this file has the encrypted private key (extracted from the extended private key), meaning that you only need the password to spend the bitcoin.
```
Comment (lontivero) [inline docs/FAQ/FAQ-UseWasabi.md]: Ok, this is correct but it is just a technical/implementation detail that can help a few people to understand better the process while confuse the majority of the users. What is the point to explain that the private key is the one from the extended private key? 

Imo I would just say "... has the encrypted private key", this is better than saying "private keys" what is simply not true.

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md]: > What is the point to explain that the private key is the one from the extended private key?

Most of the people know about a private key related to one address which is important, but the private key mentioned in this sentence is not the same one and is more important.
(Just a clarification, and if someone is willing to know they might do some research)

Comment (MaxHillebrand) [reply docs/FAQ/FAQ-UseWasabi.md]: I am in favor of accuracy, even if it comes at the expense of confusing some users... 
I don't like talking down to users, I assume they are willing enough to accumulate difficult but accurate knowledge and try to understand it. 

So I ACK these changes

**Comment by RiccardoMasutti** [comment]
This should close #145 right? @yahiheb

**Comment by yahiheb** [comment]
No it shouldn't.

**Pull request merged**